stdlib/2and3/logging/__init__.pyi

```diff
@@ -22,6 +22,7 @@ _Level = Union[int, Text]
 
 raiseExceptions: bool
 
+_levelNames = ...  # type: Dict[int, str]
```

Comment:
In Python 3.6, this doesn't exist; instead there is `_levelToName`. In Python 2.7 it does exist under this name, but it has both the int/str mapping and the reverse str/int mapping. I haven't checked earlier versions of Python 3, but do we really need this private name?